Angular/src/app/maintenance/maintenance/maintenance-type/maintenance-type.component.ts

```diff
@@ -0,0 +1,87 @@
+  import { AfterViewInit, Component, OnInit, ViewChild } from '@angular/core';
+  import { MatPaginator } from '@angular/material/paginator';
+  import { MatSnackBar, MatSnackBarRef } from '@angular/material/snack-bar';
+  import { MatSort } from '@angular/material/sort';
+  import { MatTableDataSource } from '@angular/material/table';
+  import { Router } from '@angular/router';
+  import { Observable } from 'rxjs';
+  import { MaintenanceService } from 'src/app/services/maintenance.service';
+  import { Snaglistitem } from 'src/app/shared/SnagListItem';
```

Comment:
How is this import working? There is no class in shared folder called SnagListItem?